### DISCUSSION TITLE
Migrating away from jose-elias-alvarez/typescript.nvim

### DISCUSSION THREAD

**Discussion by AndersNS** [LazyVim/LazyVim#1449] [Q&A]
Hey. 

So I have encountered quite a bit of issues with typescript lately. Sometimes my editor freezes completely when opening some files. I disabled the typescript plugin and it started working again, so it is something with the LSP setup I think. 

I moved to the [typescript-tools.nvim](https://github.com/pmizio/typescript-tools.nvim) plugin and its working again, but I dont like the way typescript-tools sets up the LSP. Does anyone have a nice solution for this? Or a more manual LSP-setup for typescript they can share? :) 

Thanks

**Comment by itsjoeoui** [comment]
+1 on this

**Comment by redbeardymcgee** [comment]
[My solution](https://github.com/redbeardymcgee/dotfiles/blob/e5c0ac566277d6bf08e798bee48b05529d8d09a0/dot_config/nvim/lua/plugins/typescript.lua) 

First -  `:Mason uninstall typescript-language-server`

**Comment by rubiin** [comment]
Heres mine https://github.com/rubiin/init.lua/blob/914259fdb477733ebf78d347e185542aca68bcda/lua/plugins/extras/lang/typescript.lua#L49
